docs/data/install-xroot-client.md

```diff
@@ -1,97 +1,196 @@
+client RPM.
+Mounting a XRootD data store using FUSE will require installing the XRootD FUSE rpm and the client rpm.
+
+### Installing the XRootD Client RPM
+
+The XRootD client rpm is needed to access XRootD using `xrdcp` or POSIX preload.  
+The following steps will install the software on your system.
+
+1. Clean yum cache:
+
+        ::console
+        root@host # yum clean all --enablerepo=\*
+
+1. Update software:
+
+        :::console
+        root@host # yum update
+
+    This command will update **all** packages
+
+1. Install XRootD Client rpm:
+
+        :::console
+        root@host # yum install xrootd-client
+
+
+### Installing the XRootD FUSE RPM
+
+The XRootD FUSE rpm is only needed if you are planning on mounting a local XRootD data store.
+If you are planning on using a FUSE mount, you'll need to install the client rpm as outlined above and also run the following command:
+
+1. Install XRootD FUSE rpm:
+
+        :::console
+        root@host # yum install xrootd-fuse
+
+
+Configuring the XRootD Client Software
+--------------------------------------
+
+Once the appropriate RPM(s) are installed, only the FUSE setup will need further configuration.
+See [this](/docs/data//install-xrootd#optional-enabling-a-fuse-mount) for instructions on updating your `fstab` file.
+
+
+Using the XRootD Client Software
+----------------------------------
+
+### Using the XRootD client tools
+
+Once the `xrootd-client` rpm is installed, you should be able to use the `xrdcp` command to copy files to and from
+XRootD systems and the local file system.
+For example:
 
 ``` console
-root@host # yum install xrootd-client
+user@host # echo "This is a test" >/tmp/test 
+user@host # xrdcp /tmp/test xroot://redirector.domain.org:1094//storage/path/test 
+user@host # xrdcp xroot://redirector.domain.org:1094//storage/path/test /tmp/test1 
+user@host # diff /tmp/test1 /tmp/test 
 ```
 
-### Installing XRootD Fuse
-
-The XRootD fuse package is needed to access via XRootDFS.
+For other operations, you'll need to use the `xrdfs` command.
+This command allows you to do file operations such as creating directories, removing directories, deleting files, and
+moving files on a XRootD system, provided you have the appropriate authorization.
+The `xrdfs` command can be used interactively by running `xrdfs xroot://redirector.domain.org:1094/`.
+Alternatively, you can use it in batch mode by adding the `xrdfs` command after the xroot URI.  For example:
 
 ``` console
-root@host # yum install xrootd-fuse
+user@host # echo "This is a test" >/tmp/test 
+user@host # xrdfs xroot://redirector.domain.org:1094/  mkdir  /storage/path/test
+user@host # xrdcp xroot://redirector.domain.org:1094//storage/path/test/test1 /tmp/test1 
+user@host # xrdfs xroot://redirector.domain.org:1094/  ls  /storage/path/test/test1
+user@host # xrdfs xroot://redirector.domain.org:1094/  rm  /storage/path/test/test1
+user@host # xrdfs xroot://redirector.domain.org:1094/  rmdir  /storage/path/test
```

Comment:
Let's replace `user@host #` with `user@client $` to indicate that this can happen from another host